priorityq.py

```diff
@@ -0,0 +1,38 @@
+#! /usr/bin/env python
+# Priority Queue Python file for Efrain, Mark and Henry
+import queue as q
+
+
+class Pq(object):
+    """Initiate priority queue"""
+
+    def __init__(self, num=3):
+        """Initiate pq with optional priority range, default is 3."""
+        for x in range(1, num+1):
+            setattr(self, 'que{}'.format(x), q.Queue())
+        self.num_queues = num
```

Comment:
What about basing the entire structure on something you built before?  What other data structures do you have that maintain an ordering as you insert and remove items?